gee/utils.py

```diff
@@ -967,7 +967,7 @@ def listToObject(values):
         .get('list') \
         .getInfo()
 
-    collectionBands = map(listToObject, collectionBands)

```

Comment:
dont re-asign a variable to itself.  You can return the value directly. `return list(map(listToObject, collectionBands))`